packages/lambda-powertools-middleware-correlation-ids/index.js

```diff
@@ -1,298 +1,25 @@
-
-        const correlationIdsInstance = new CorrelationIds(correlationIds)
-
-        Object.defineProperties(event, {
-          correlationIds: {
-            value: correlationIdsInstance,
-            enumerable: false
-          },
-          logger: {
-            value: new Log({ correlationIds: correlationIdsInstance }),
-            enumerable: false
-          }
-        })
-
-        return event
-      } catch (e) {
-        return initCorrelationIds(context, sampleDebugLogRate)
-      }
-    })
-
-  context.parsedKinesisEvents = events
-
-  // although we're going to have per-record correlation IDs, the default one for the function
-  // should still have the awsRequestId at least
-  CorrelationIds.replaceAllWith({
-    'x-correlation-id': awsRequestId,
-    awsRequestId,
-    [DEBUG_LOG_ENABLED]: Math.random() < sampleDebugLogRate ? 'true' : 'false'
-  })
-}
-
-function captureContextField ({ __context__ }, { awsRequestId }, sampleDebugLogRate) {
-  const correlationIds = __context__ || {}
-  correlationIds.awsRequestId = awsRequestId
-  if (!correlationIds[X_CORRELATION_ID]) {
-    correlationIds[X_CORRELATION_ID] = awsRequestId
-  }
-
-  if (!correlationIds[DEBUG_LOG_ENABLED]) {
-    correlationIds[DEBUG_LOG_ENABLED] = Math.random() < sampleDebugLogRate ? 'true' : 'false'
-  }
-
-  correlationIds[CALL_CHAIN_LENGTH] = (correlationIds[CALL_CHAIN_LENGTH] || 0) + 1
-
-  CorrelationIds.replaceAllWith(correlationIds)
-}
-
-function initCorrelationIds ({ awsRequestId }, sampleDebugLogRate) {
-  const correlationIds = { awsRequestId }
-  correlationIds[X_CORRELATION_ID] = awsRequestId
-  correlationIds[DEBUG_LOG_ENABLED] = Math.random() < sampleDebugLogRate ? 'true' : 'false'
-  correlationIds[CALL_CHAIN_LENGTH] = 1
-
-  CorrelationIds.replaceAllWith(correlationIds)
-}
-
-function isApiGatewayEvent (event) {
-  return event.hasOwnProperty('httpMethod')
-}
-
-function isSnsEvent (event) {
-  if (!event.hasOwnProperty('Records')) {
-    return false
-  }
-
-  if (!Array.isArray(event.Records)) {
-    return false
-  }
-
-  return event.Records[0].EventSource === 'aws:sns'
-}
-
-function isSqsEvent (event) {
-  if (!event.hasOwnProperty('Records')) {
-    return false
-  }
-
-  if (!Array.isArray(event.Records)) {
-    return false
-  }
-
-  return event.Records[0].eventSource === 'aws:sqs'
-}
-
-function isKinesisEvent (event) {
-  if (!event.hasOwnProperty('Records')) {
-    return false
-  }
-
-  if (!Array.isArray(event.Records)) {
-    return false
-  }
-
-  return event.Records[0].eventSource === 'aws:kinesis'
-}
-
-function hasContextField (event) {
-  return event.hasOwnProperty('__context__')
-}
+const generic = require('./event_sources/generic')
+const eventSources = [
+  require('./event_sources/api_gateway'),
+  require('./event_sources/sns'),
+  require('./event_sources/sqs'),
+  require('./event_sources/kinesis'),
+  require('./event_sources/direct_invoke')
```

Comment:
other folders/files in this project follow `kebab-case` (personal preference of mine too)

Reply:
good catch! we should be internally consistent

Reply:
done